### PR TITLE
Fixed wrong placeholder for total xp levels spent.

### PR DESCRIPTION
**Pull request by variananora**
Changing the wiki from 
`%slimefun_total_xp_levels_spent%`
to 
`%slimefun_researches_total_xp_levels_spent%`

### CODE REVIEW

**Review by svr333 — DISMISSED** [review]
lgtm

**Comment by TheBusyBiscuit** [comment]
Please look into branches:
https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-branches#working-with-branches

This way, you can make multiple pull requests and keep the changes seperate from each other.
It is best practice to keep a PR as small as possible, as it makes the reviewing a whole lot easier and changes can be applied to limited scopes.

File: pages/Programmable-Android.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+This interface is used to store items from an Android when the script tells then to do so.
+This interface is also can be connected through a cargo network or with a hopper.
+
+### Android Interfaces (Fuel)
+
+Recipe:
+
+This interface is used to give Android the fuel from it when the script tells then to do so.
+This interface is also can be connected through a cargo network or with a hopper.
```
Comment (TheBusyBiscuit) [inline pages/Programmable-Android.md:17]: I would argue that the Interfaces deserve their own page

Comment (variananora) [reply pages/Programmable-Android.md:17]: > Please look into branches:
> https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-branches#working-with-branches
> 
> This way, you can make multiple pull requests and keep the changes seperate from each other.
> It is best practice to keep a PR as small as possible, as it makes the reviewing a whole lot easier and changes can be applied to limited scopes.

ah fuck i forgot to make a branch. I'll revert it to placeholder only for this one. (the android page is still on concept, also will change it to their own individual type page)

Comment (TheBusyBiscuit) [reply pages/Programmable-Android.md:17]: :+1:

**Comment by variananora** [comment]
Done, I'm too retarded to use GitHub, lol.
This should fix the placeholder only.

**Review by TheBusyBiscuit — APPROVED** [review]
👍🏻

**Review by svr333 — APPROVED** [review]

**Pull request merged**